FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -267,12 +267,37 @@ The nth day is an ordinal number, not a cardinal number. &apos;1&apos; means the
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>date time stamp</rdfs:label>
-		<skos:definition>a combination of a date, a time, and a time zone</skos:definition>
+		<skos:definition>combination of a date, a time, and a time zone</skos:definition>
 		<fibo-fnd-utl-av:synonym>time stamp</fibo-fnd-utl-av:synonym>
 		<fibo-fnd-utl-av:usageNote>&apos;hasDateTimeStampValue&apos; is omitted if the DateTimeStamp is not (yet) established.</fibo-fnd-utl-av:usageNote>
 		<fibo-fnd-utl-av:usageNote>Use DateTimeStamp when the time stamp includes a time zone field. Use DateTime for the combination of a Date and a time, without a time zone.</fibo-fnd-utl-av:usageNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-dt-fd;DatedCollectionConstituent">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;CollectionConstituent"/>
+		<rdfs:label>dated collection constituent</rdfs:label>
+		<owl:equivalentClass>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasObservedDateTime"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
+			</owl:Restriction>
+		</owl:equivalentClass>
+		<skos:definition>element of a collection that is associated with a date and time</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Note that the use of several options for the representation of a date and time stamp enables extensions for milliseconds, nanoseconds using an xsd:string that has the format of an xsd:dateTime datatype but extends the level of granularity consistently.  An example of where this is required is to represent prices that change multiple times in a given day.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-dt-fd;DatedStructuredCollection">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;StructuredCollection"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;DatedCollectionConstituent"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>dated structured collection</rdfs:label>
+		<skos:definition>structured collection that includes some dated elements</skos:definition>
```

Comment:
circular and vague. 
Can some constituents really not be dated? 
Can a dated constituent include a CalculatedDate which has not been calculated yet?

Reply:
I've updated this definition to reflect how we are using it, namely to identify collections whose elements are ordered by date and time.  I think if the observation date and time is missing, it's not part of a dated structured collection.  It could still be part of a structured collection, but not a dated one.